db/migrate/20230919130352_create_posts.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePosts < ActiveRecord::Migration[7.0]
+  def change
+    create_table :posts do |t|
+      t.integer :AuthorId
```

Comment:
- [ ] @Kgomotso196 Great work in your comments table, however, I would recommend the removal of this column as it will  be automatically generated when you run your migration(using add_Refernece which automatically adds the column and index to the foreign key )

Reply:
Thank you for your constructive feedback @danmainah.